getblocklab/block-lab
# Add handling for the Pro plugin

#### Changes
* On entering a valid subscription key, this downloads the Pro plugin, not the free one.

#### Testing instructions
1. If you have GCB Pro in `wp-content/plugins/`, rename it to `genesis-custom-blocks-pro-renamed/` so this can install GCB Pro from the endpoint
1. Ensure there is a valid Block Lab Pro subscription key saved
1. Go to `/wp-admin` > Migrate 
1. Enter a valid Genesis Pro subscription key
1. Expected: <img width="1022" alt="here-after-submitting" src="https://user-images.githubusercontent.com/4063887/91625117-58922780-e96a-11ea-8034-9d6fdc4ff739.png">
1. Continue through the migration flow to actually installing GCB
1. Expected: After clicking 'Install and activate,' this should actually install GCB Pro from the endpoint, and it should work
1. The version of GCB Pro it downloads from the endpoint isn't the final one that'll be released, but it's pretty close
